nebullvm/operations/optimizations/compilers/openvino.py

```diff
@@ -25,6 +25,137 @@
 from nebullvm.tools.transformations import MultiStageTransformation
 
 
+# classe di base. Contains generic methods.
+class TensorFlowOpenVINOCompiler(Compiler):
+    supported_ops = {
+        "cpu": [
+            None,
+            # QuantizationType.STATIC,
+            # QuantizationType.HALF,
+        ],
+        "gpu": [],
+    }
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(
+        self,
+        model: Union[str, Path],
+        model_params: ModelParams,
+        input_tfms: MultiStageTransformation = None,
+        metric_drop_ths: float = None,
+        quantization_type: QuantizationType = None,
+        input_data: DataManager = None,
+        **kwargs,
+    ):
+        """Compile the input model using TF-OPENVINO library.
+
+        Args:
+            model (str): The onnx model path.
+            model_params (ModelParams): The model parameters.
+            input_tfms (MultiStageTransformation, optional): Transformations
+                to be performed to the model's input tensors in order to
+                get the prediction. Default: None.
+            metric_drop_ths (float, optional): Threshold for the accepted drop
+                in terms of precision. Any optimized model with a higher drop
+                will be ignored. Default: None.
+            quantization_type (QuantizationType, optional): The desired
+                quantization algorithm to be used. Default: None.
+            input_data (DataManager): User defined data. Default: None
+        """
+
+        if quantization_type not in self.supported_ops[self.device.value]:
+            self.compiled_model = None
+            return
+
+        if quantization_type is QuantizationType.STATIC and input_data is None:
+            raise ValueError("Input data is required for static quantization.")
+
+        self.logger.info(
+            f"Optimizing with {self.__class__.__name__} and "
+            f"q_type: {quantization_type}."
+        )
+
+        check_quantization(quantization_type, metric_drop_ths)
+        train_input_data = input_data.get_split("train").get_numpy_list(
+            QUANTIZATION_DATA_NUM
+        )
+
+        #   SYNTAX for MO command
+        #        f"""mo
+        #                 --saved_model_dir "{model_path}"
+        #                 --input_shape "[1,224,224,3]"
+        #                 --mean_values="[127.5,127.5,127.5]"
+        #                 --scale_values="[127.5]"
+        #                 --model_name "{model_path.name}"
+        #                 --compress_to_fp16
+        #                 --output_dir "{model_path.parent}"
+        #                 """
```

Comment:
remove the commented code above

Reply:
deleted!

Reply:
I don't see any difference. Did you push the code?